README.md

```diff
@@ -2,6 +2,11 @@ Visit [the official web site](https://line.github.io/centraldogma/) for more inf
 
 # Central Dogma
 
+[![CI](https://github.com/line/centraldogma/actions/workflows/actions_build.yml/badge.svg?branch=main&event=push)](https://github.com/line/centraldogma/actions/workflows/actions_build.yml)
+[![codecov.io](https://codecov.io/github/line/centraldogma/coverage.svg?branch=main)](https://codecov.io/github/line/centraldogma?branch=main)
+[![Latest Release Version](https://img.shields.io/github/v/release/line/centraldogma)](https://github.com/line/centraldogma/releases/latest)
+[![Slack workspace](https://img.shields.io/badge/join-slack-orange?logo=slack)](https://join.slack.com/t/central-dogma/shared_invite/enQtNjA5NDk5MTExODQzLWFhOWU2NGZhNDk3MjBmNzczZDYyZjRmMTI1MzdiNGI3OTcwNWZlOTkyY2U3Nzk4YTM2NzQ2NGJhMjQ1NzJlNzQ)
```

Comment:
We have moved to Discord: https://armeria.dev/s/discord
Could you fix that, please? 😄

Reply:
Fixed :)